lib/bandwidth/voice_lib/bxml/verbs/record.rb

```diff
@@ -19,6 +19,7 @@ def to_bxml(xml)
          'maxDuration' => max_duration,
          'fileFormat' => file_format,
          'transcribe' => transcribe,
+         'detectLanguage' => detectLanguage,
```

Comment:
```suggestion
         'detectLanguage' => detect_language,
```
ruby uses snake_case for variable names, the instances referring to the `detectLanguage` variable should be changed to `detect_language`